src/component/ResponsiveContainer.tsx

```diff
@@ -162,6 +164,7 @@ export const ResponsiveContainer = forwardRef<HTMLDivElement, Props>(
         style={{ ...style, width, height, minWidth, minHeight, maxHeight }}
         ref={containerRef}
       >
+        <style>{`.recharts-wrapper > .recharts-surface {width: 100%; height: 100%;}`}</style>
```

Comment:
I don't think its a good idea to have a style tag in the body of the document. Is there any way we can do this another way?

Reply:
I removed the style tags and added inline styles.